extras/uwsgi/uwsgi.service

```diff
@@ -1,13 +0,0 @@
-[Unit]
-Description=uWSGI Emperor
-
-[Service]
-ExecStart=/usr/bin/uwsgi --emperor /etc/uwsgi/apps-enabled --logto /var/log/uwsgi/access.log
-RuntimeDirectory=uwsgi
-Restart=always
-KillSignal=SIGQUIT
-Type=notify
-NotifyAccess=all
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
What will start the service then ? Is there any sysvinit script that we can rely on ? 

I'm using this unit to start other services working with uwsgi, plus I use the "after" options to start Uwsgi service after MongoDB, this is pretty efficient.